src/util/EditionCardUtils.tsx

```diff
@@ -304,4 +343,13 @@ export default class EditionCardUtils {
       );
     }
   }
+
+  // Get any link except for catalog
+  static getPreviewItem(items: ApiItem[] | undefined) {
```

Comment:
In the Search Results page, does this method get called, or does it first apply the filter?  eg:  When you filter for "PDFs only", the preview items should be the ones with PDFs.

Reply:
This seems fine because it performs the search filter before calling the method.